api/types/github.go

```diff
@@ -218,8 +242,8 @@ func (c *GithubConnectorV3) SetDisplay(display string) {
 
 // MapClaims returns a list of logins based on the provided claims,
 // returns a list of logins and list of kubernetes groups
-func (c *GithubConnectorV3) MapClaims(claims GithubClaims) ([]string, []string, []string) {
-	var logins, kubeGroups, kubeUsers []string
+func (c *GithubConnectorV3) MapClaims(claims GithubClaims) ([]string, []string, []string, []string) {
```

Comment:
Well that was a pretty wierd trip down legacy-lane. Turns out, you're right. We don't map the logins field to *nix logins even though we have both a roles and logins field in our `createUserParams` and have code + tests feeding it login as well as roles values and a bunch of comments referring to it. Looks like we do have all the code needed to do it but we just discard that field last minute before creating a user.

I've gotten rid of that field and simplified mapclaims and removed/renamed anything confusing so this doesn't happen again.

> Further, GetTeamsToLogins and SetTeamsToLogins could just have the same body as their TeamsToRoles variant, making them easier to deprecate.

This I am not so sure about. That is because we do set kubegroups etc as traits that we pass on which means those fields are still used, thus, the mappings still aren't totally equivalent, so it looks like I need to keep `GetTeamsToLogins` and `SetTeamsToLogins` as-is.